apps/teams-test-app/e2e-test-data/meeting.json

```diff
@@ -35,11 +35,11 @@
     },
     {
       "title": "getMeetingDetailsVerbose API Call - Success",
-      "version": ">2.22.0",
```

Comment:
rather than edit this test, you will want to modify the version to be `> 2.22.0 && <= 2.28.0` and then leave the expected value as is. You can then introduce a new test with version `> 2.28.0` that has your new expected value

Reply:
Updated test, thanks!